commands/commands.go

```diff
@@ -0,0 +1,73 @@
+package commands
+
+import (
+	"github.com/bwmarrin/discordgo"
+	"github.com/bwmarrin/lit"
+)
+
+// Version is a constant that stores the Disgord version information.
+const Version = "v0.1.0-rewrite"
+
+var (
+	AdminUserID  string // Skippy
+	HerderRoleID string
+	Commands     = make(map[string]*Command)
+)
+
+type Command struct {
+	*discordgo.ApplicationCommand
+	Handler func(*discordgo.Session, *discordgo.InteractionCreate) (*discordgo.InteractionResponseData, error)
+}
+
+func OnInteractionCommand(ds *discordgo.Session, ic *discordgo.InteractionCreate) {
+	if ic.Type != discordgo.InteractionApplicationCommand {
+		return
+	}
+
+	data := ic.ApplicationCommandData()
+	cmd, ok := Commands[data.Name]
+	if !ok {
+		return
+	}
+
+	res, err := cmd.Handler(ds, ic)
+	if err != nil {
+		res = &discordgo.InteractionResponseData{
+			Flags: 64,
```

Comment:
Consider making 64 a constant. Might be worth PRing the constant into discordgo too (there are missing MessageFlags).